apps/E2E/src/ButtonExperimental/specs/ButtonExperimental.spec.win.ts

```diff
@@ -22,55 +21,78 @@ describe('Experimental Button Testing Initialization', function () {
 });
 
 describe('Experimental Button Accessibility Testing', async () => {
+  beforeEach(async () => {
+    await ButtonExperimentalPageObject.scrollToTestElement(await ButtonExperimentalPageObject._primaryComponent);
```

Comment:
I don't think you need to specify `await ButtonExperimentalPageObject._primaryComponent` since `scrollToTestElement` will scroll to the _primaryComponent if no component is specified.